.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+### Summary
+[PLEASE REMOVE] Describe the goal of this PR and add any relevant context.
```

Comment:
I think this is a good place to call out our detailed guidelines. Most devs won't think to look there, but this is a chance to raise awareness.

```suggestion
### Summary
[PLEASE REMOVE] See the [Pull Requests section of CONTRIBUTING.md](https://github.com/pytorch/executorch/blob/main/CONTRIBUTING.md#pull-requests) for ExecuTorch PR guidelines.

[PLEASE REMOVE] Describe the goal of this PR and add any relevant context.
```